sonar-plugin-api/src/main/java/org/sonar/api/batch/DecoratorContext.java

```diff
@@ -88,17 +87,13 @@
   DecoratorContext saveMeasure(Metric metric, Double value);
 
   // DEPENDENCIES
-
+  /**
+   * @deprecated since 5.2 No more design features. No-op.
+   */
+  @Deprecated
   Dependency saveDependency(Dependency dependency);
```

Comment:
:new: ![INFO](https://raw.githubusercontent.com/henryju/image-hosting/master/info.png)Do not forget to remove this deprecated code someday.[![...](https://raw.githubusercontent.com/henryju/image-hosting/master/more.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1133)